datadotworld/client/api.py

```diff
@@ -441,6 +445,314 @@ def download_datapackage(self, dataset_key, dest_dir):
+            Maximum number of items to include in a page of results
+        next : str, optional
+            Token from previous result page (to be used when requesting a subsequent page)
+        sort : str, optional
+            Property name to sort
+
+        Returns
+        -------
+        dict
+            Authenticated user dataset
+
+        Raises
+        ------
+        RestApiException
+            If a server error occurs
+
+        Examples
+        --------
+        >>> import datadotworld as dw
+        >>> api_client = dw.api_client()
+        >>> user_dataset = api_client.fetch_contributing_datasets()
+        {'count': 0, 'records': [], 'next_page_token': None}
+        """
+        try:
+            return self._user_api.fetch_contributing_datasets()
+        except _swagger.rest.ApiException as e:
+            raise RestApiError(cause=e)
+
+    def fetch_liked_datasets(self):
+        """Fetch datasets that authenticated user likes
+
+        Parameters
+        ----------
+        limit : str, optional
+            Maximum number of items to include in a page of results
+        next : str, optional
+            Token from previous result page (to be used when requesting a subsequent page)
+        sort : str, optional
+            Property name to sort
+
+        Returns
+        -------
+        dict
+            Dataset definition, with all attributes
+
+        Raises
+        ------
+        RestApiException
+            If a server error occurs
+
+        Examples
+        --------
+        >>> import datadotworld as dw
+        >>> api_client = dw.api_client()
+        >>> user_liked_dataset = api_client.fetch_liked_datasets()
+        >>> user_liked_dataset.count
+        1
+        """
+        try:
+            return self._user_api.fetch_liked_datasets()
+        except _swagger.rest.ApiException as e:
+            raise RestApiError(cause=e)
+
+    def fetch_datasets(self):
+        """Fetch authenticated user owned datasets
+
+        Parameters
+        ----------
+        limit : str, optional
+            Maximum number of items to include in a page of results
+        next : str, optional
+            Token from previous result page (to be used when requesting a subsequent page)
+        sort : str, optional
+            Property name to sort
+
+        Returns
+        -------
+        dict
+            Dataset definition, with all attributes
+
+        Raises
+        ------
+        RestApiException
+            If a server error occurs
+
+        Examples
+        --------
+        >>> import datadotworld as dw
+        >>> api_client = dw.api_client()
+        >>> user_owned_dataset = api_client.fetch_datasets()
+        >>> user_owned_dataset.records
+        []
+        """
+        try:
+            return self._user_api.fetch_datasets()
+        except _swagger.rest.ApiException as e:
+            raise RestApiError(cause=e)
+
+    # Sql Operations
+
+    def sql_get(self, dataset_key, query):
```

Comment:
There is no need to expose both `sql_get` and `sql_post` at this level. Please remove `sql_get` and rename `sql_post` to `sql`.